media/css/mozorg/home/home.scss

```diff
@@ -397,13 +388,14 @@ main {
     }
 }
 
-// float layout for wider viewports
+// flex/float layout for wider viewports
 @media #{$mq-tablet} {
-    #who-innovate-wrapper {
+    #impact-innovate-wrapper {
         @include clearfix;
+        @include flexbox;
```

Comment:
Good idea, wish I'd thought of it! Changed to fake table layout, which should work in IE9/10 as well.